src/plugins/interactive_setup/public/doc_link.tsx

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import type { EuiLinkAnchorProps } from '@elastic/eui';
+import { EuiLink } from '@elastic/eui';
+import type { FunctionComponent } from 'react';
+import React, { useCallback } from 'react';
+
+import type { CoreStart } from 'src/core/public';
+
+import { useKibana } from './use_kibana';
+
+export type DocLinks = CoreStart['docLinks']['links'];
+export type GetDocLinkFunction = (app: string, doc: string) => string;
+
+/**
+ * Creates links to the documentation.
+ *
+ * @see {@link DocLink} for a component that creates a link to the docs.
+ *
+ * @example
+ * ```typescript
+ * <DocLink app="elasticsearch" doc="built-in-roles.html">
+ *   Learn what privileges individual roles grant.
+ * </DocLink>
+ * ```
+ *
+ * @example
+ * ```typescript
+ * const [docs] = useDocLinks();
+ *
+ * <EuiLink href={docs.dashboard.guide} target="_blank" external>
+ *   Learn how to get started with dashboards.
+ * </EuiLink>
+ * ```
+ */
+export function useDocLinks(): [DocLinks, GetDocLinkFunction] {
```

Comment:
__issue:__ if I remember correctly, we're supposed to keep all links directly in the Core's `doc_links_service` to make it easier for us and the Docs Team (they own and maintain this list whenever doc URL structure changes or docs are moved around)?

Reply:
Will leave as is for now until we know what it should be changed to.

Reply:
As per https://github.com/elastic/kibana/pull/113011#issuecomment-933850686, It seems we'll be using this link: https://www.elastic.co/guide/en/elasticsearch/reference/master/configuring-stack-security.html (the one that you picked initially I believe)?